Source/AppLocker/AppLocker.swift

```diff
@@ -118,30 +121,31 @@ public class AppLocker: UIViewController {
             isFirstCreationStep = false
             reservedPin = pin
             clearView()
-            submessageLabel.text = "Confirm your pincode"
+            submessageLabel.text = "Confirm pin"
```

Comment:
This should probably be "PIN" as it's an abbreviation.  Localization as well

Reply:
Sure